### PR TITLE
Issue #19 feat: Delete question option to be enabled

### PR DESCRIPTION
**Pull request by AkshayaRathinavel**
https://project-sunbird.atlassian.net/browse/SB-4910

### CODE REVIEW

File: editor/questionbankcontroller.js [hunk 1]
```diff
@@ -472,6 +472,57 @@ angular.module('createquestionapp', [])
         ecEditor.dispatchEvent($scope.pluginIdObj.question_create_id + ":showpopup", questionObj);
       }
     }
+
+    $scope.deleteQuestion = function(questionObj){
+      $scope.assessmentId = questionObj.IL_UNIQUE_ID;
```
Comment (manojchandrashekar) [inline editor/questionbankcontroller.js]: I don't remember using this reference before `questionObj.IL_UNIQUE_ID`. Is this the same as `questionObj.identifier`?

File: test/mocks/ediotor/eventListener-mock.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-windowmock.open = function(url, target, settings) {
```
Comment (manojchandrashekar) [inline test/mocks/ediotor/eventListener-mock.js:1]: The folder name is incorrect. Change it to `editor`.

**Review by manojchandrashekar — CHANGES_REQUESTED** [review]

**Comment by vinukumar-vs** [comment]
https://project-sunbird.atlassian.net/browse/SB-4910
Scenarios are not updated in JIRA issue. Hence closing PR

File: editor/questionbankcontroller.js [hunk 3]
```diff
@@ -530,6 +530,57 @@ angular.module('createquestionapp', [])
...
+      } else {
+        ecEditor.dispatchEvent("org.ekstep.toaster:error", {
+          title: 'Failed to delete question...',
+          position: 'topCenter',
+        });
+      }
+      $scope.$safeApply();
+    }
+
+    $scope.deleteQuestionHandler = function(questionObj) {
```
Comment (vinukumar-vs) [inline editor/questionbankcontroller.js:557]: This logic has to improve. Too much code.

**Review by vinukumar-vs — APPROVED** [review]

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**